libs/ssl-config/src/main/java/org/elasticsearch/common/ssl/SslConfiguration.java

```diff
@@ -58,14 +67,6 @@
         ORDERED_PROTOCOL_ALGORITHM_MAP = Collections.unmodifiableMap(protocolAlgorithmMap);
     }
 
-    private final boolean explicitlyConfigured;
```

Comment:
I guess not having this in the equals and hashcode was a bug before. This might be behaviour change.

Reply:
:+1: Let me bring back `equals&hashCode` in order not to introduce unwanted changes